processor/src/main/resources/org/mapstruct/ap/internal/model/MethodReference.ftl

```diff
@@ -20,7 +20,16 @@
 -->
 <@compress single_line=true>
     <#-- method is either internal to the mapper class, or external (via uses) declaringMapper!=null -->
-    <#if declaringMapper??><#if static><@includeModel object=declaringMapper.type/><#else>${mapperVariableName}</#if>.</#if>${name}<#if (parameters?size > 0)>( <@arguments/> )<#else>()</#if>
+    <#if declaringMapper??><#if static><@includeModel object=declaringMapper.type/><#else>${mapperVariableName}</#if>.<@params/>
+    <#elseif static && mapperToImplement??><@includeModel object=mapperToImplement/>.<@params/>
```

Comment:
> Does this work with a multi-leveled inheritance chain, too?

No.. It doesn't.. Added a test and it fails (for the reason it cannot find the static method).  Need to look in to that more carefully..  I'll finish it this evening..